twilio-flex-webchat/src/components/DailyVideo.jsx

```diff
@@ -0,0 +1,139 @@
+import * as FlexWebChat from "@twilio/flex-webchat-ui";
+import { useCallback, useEffect, useRef, useState } from "react";
+import DailyIframe from "@daily-co/daily-js";
+import api from "../api";
+/**
+ * daily-js call properties: https://docs.daily.co/reference/daily-js/daily-iframe-class/properties
+ */
+const CALL_OPTIONS = {
+  iframeStyle: {
+    width: "100%",
+    height: "100%",
+    maxWidth: "275px",
+    aspectRatio: 9 / 16,
+    border: "1px solid #e6eaef",
+    borderRadius: "6px 6px 0 0",
+    boxShadow: `0 1px 2px rgba(0, 0, 0, 0.02), 0 2px 4px rgba(0, 0, 0, 0.02),
+      0 4px 8px rgba(0, 0, 0, 0.02), 0 8px 16px rgba(0, 0, 0, 0.02),
+      0 16px 32px rgba(0, 0, 0, 0.02)`,
+  },
+  showLeaveButton: false,
```

Comment:
I'm sure you were already going to explain all these in the blog post, but this is one that could be worth calling out. It took me longer than I'd like to admit to notice the ❌ , 🤦‍♀️

Reply:
maybe I'll update that button to be more obvious then, thanks!